src/blocks/blocks/accordion/group/inspector.js

```diff
@@ -87,6 +87,17 @@ const Inspector = ({
 		setAttributes({ tag: targetTag });
 	};
 
+	const onBorderChange = ( type, property, value ) => {
+		if ( ! [ 'header', 'content' ].includes( type ) ) {
+			return;
+		}
+
+		const border = ( 'header' === type ? attributes.headerBorder : attributes.contentBorder ) || {};
+
+		border[property] = 'object' === typeof value ? { ...border[property], ...value } : value;
+		setAttributes({ [ type + 'Border' ]: { ...border } });
+	};
+
```

Comment:
You do not need to encode the props with `header`, `content` because this is not a library that tries to hide the implementation from the user. Since is only for internal use, you can go directly with the name of the state.

You can use this:

```javascript
	const onBorderChange = ( attrName, property, value ) => {
		setAttributes({ 
			[ attrName ]: {...attributes[attrName], [property]: 'object' === typeof value ? { ...attributes[attrName][property], ...value } : value} 
		});
	};
```

An have a call like: `value => onBorderChange( 'headerBorder', 'color', value )`